nil/specification/data-availability.mdx

```diff
@@ -0,0 +1,140 @@
+# Data availability
+
+## Definition
+
+In =nil;, data availability (DA) is ensured via the following methods:
+
+* The synchronization committee (operating on top of the consensus shard) submits DA transactions to Ethereum
+* The consensus shard submits snapshots of the global state as state diffs to Ethereum  
```

Comment:
The consensus shard cannot do it. All data is submitted by sync committee. Ping Ilya Marozau for additional information

Reply:
I see, I have simplified this page slightly after a convo with Ilya Marozau: we decided to only provide info about the sync committee that would be relevant to devs